docs/cheatsheet.rst

```diff
@@ -13,68 +13,108 @@ Order of Precedence of Operators
-- ``super``: the contract one level higher in the inheritance hierarchy
-- ``selfdestruct(address payable recipient)``: destroy the current contract, sending its funds to the given address
-- ``<address>.balance`` (``uint256``): balance of the :ref:`address` in Wei
-- ``<address>.code`` (``bytes memory``): code at the :ref:`address` (can be empty)
-- ``<address>.codehash`` (``bytes32``): the codehash of the :ref:`address`
-- ``<address payable>.send(uint256 amount) returns (bool)``: send given amount of Wei to :ref:`address`,
-  returns ``false`` on failure
-- ``<address payable>.transfer(uint256 amount)``: send given amount of Wei to :ref:`address`, throws on failure
-- ``type(C).name`` (``string``): the name of the contract
-- ``type(C).creationCode`` (``bytes memory``): creation bytecode of the given contract, see :ref:`Type Information<meta-type>`.
-- ``type(C).runtimeCode`` (``bytes memory``): runtime bytecode of the given contract, see :ref:`Type Information<meta-type>`.
-- ``type(I).interfaceId`` (``bytes4``): value containing the EIP-165 interface identifier of the given interface, see :ref:`Type Information<meta-type>`.
-- ``type(T).min`` (``T``): the minimum value representable by the integer type ``T``, see :ref:`Type Information<meta-type>`.
-- ``type(T).max`` (``T``): the maximum value representable by the integer type ``T``, see :ref:`Type Information<meta-type>`.
+.. list-table:: Global Variables
+   :widths: 50 70
+   :header-rows: 1
+
+   * - Global Variable
+     - Description
+   * - ``abi.decode(bytes memory encodedData, (...)) returns (...)``
+     - :ref:`ABI <ABI>`-Decodes the provided data. The types are given in parentheses as second argument.
+   * - ``abi.encode(...) returns (bytes memory)``
+     - :ref:`ABI <ABI>`-Encodes the given arguments
+   * - ``abi.encodePacked(...) returns (bytes memory)``
+     - Performs :ref:`packed encoding <abi_packed_mode>` of the given arguments. Note that this encoding can be ambiguous!
+   * - ``abi.encodeWithSelector(bytes4 selector, ...) returns (bytes memory)``
+     - :ref:`ABI <ABI>`-Encodes the given arguments starting from the second and prepends the given four-byte selector
+   * - ``abi.encodeCall(function functionPointer, (...)) returns (bytes memory)``
+     - ABI-encodes a call to ``functionPointer`` with the arguments found in the tuple. Performs a full type-check, ensuring the types match the function signature. Result equals ``abi.encodeWithSelector(functionPointer.selector, (...))``
+   * - ``abi.encodeWithSignature(string memory signature, ...) returns (bytes memory)``
+     -  Equivalent to ``abi.encodeWithSelector(bytes4(keccak256(bytes(signature)), ...)``
+   * - ``bytes.concat(...) returns (bytes memory)``
+     - :ref:`Concatenates variable number of arguments to one byte array<bytes-concat>`
+   * - ``string.concat(...) returns (string memory)``
+     - :ref:`Concatenates variable number of arguments to one string array<string-concat>`   
```

Comment:
```suggestion
     - :ref:`Concatenates variable number of arguments to one string array<string-concat>`
```
You've got quite a few trailing whitespaces; please take a look at the `chk_coding_style` job to see where they are.